app/models/lunch.rb

```diff
@@ -24,35 +24,35 @@ def einstein
       rest_name = 'Einstein 🍨' if Date.today.friday?
       begin
         menu = Nokogiri.HTML(open(url))
-        price = 80
         week = menu.css('h2.lunch-titel').first.content.scan(/\d/).join('').to_i
         meals = menu.css('div.field-day').select{|day| valid_date?(week, day) }.flat_map do |day|
           day.css('p').to_a.reject{|m| invalid_meal?(m) }.map do |meal|
             content = meal.content.gsub(/[\s\u00A0]/, ' ').strip
-            { title: '', summary: content, price: price } unless content.empty?
+            tag_food(content) unless content.empty?
           end.compact
         end
-        [{ name: rest_name, meals: meals }]
+        [{ name: rest_name, meals: meals, location: "Johanneberg" }]
       rescue
-        [{ name: rest_name, meals: {}}]
-      end.sort_by { |r| r[:name] }
+        []
+      end
     end
 
     def chalmrest
       date = Time.new.strftime("%Y-%m-%d")
       locale = I18n.locale.to_s
-      restaurants = {
-        "Linsen"            => "http://intern.chalmerskonferens.se/view/restaurant/linsen/RSS%20Feed.rss?today=true&locale=#{locale}",
-        "Kårrestaurangen"   => "http://intern.chalmerskonferens.se/view/restaurant/karrestaurangen/Veckomeny.rss?today=true&locale=#{locale}",
-        "L's kitchen"       => "http://intern.chalmerskonferens.se/view/restaurant/l-s-kitchen/Projektor.rss?today=true&locale=#{locale}",
-        "Express"           => "http://intern.chalmerskonferens.se/view/restaurant/express/V%C3%A4nster.rss?today=true&locale=#{locale}",
-        #"J.A Pripps"        => "http://intern.chalmerskonferens.se/view/restaurant/j-a-pripps-pub-cafe/RSS%20Feed.rss?today=true&locale=#{locale}",
-        #"Restaurang Hyllan" => "http://intern.chalmerskonferens.se/view/restaurant/hyllan/RSS%20Feed.rss?today=true&locale=#{locale}",
-        "L's Resto"         => "http://intern.chalmerskonferens.se/view/restaurant/l-s-resto/RSS%20Feed.rss?today=true&locale=#{locale}",
-        "Kokboken"          => "http://intern.chalmerskonferens.se/view/restaurant/kokboken/RSS%20Feed.rss?today=true&locale=#{locale}"}
+      restaurants = [
+        {name: "Linsen", url: "http://intern.chalmerskonferens.se/view/restaurant/linsen/RSS%20Feed.rss?today=true&locale=#{locale}", location: "Johanneberg"},
+        {name: "Kårrestaurangen", url: "http://intern.chalmerskonferens.se/view/restaurant/karrestaurangen/Veckomeny.rss?today=true&locale=#{locale}", location: "Johanneberg"},
+        {name: "L's kitchen", url: "http://intern.chalmerskonferens.se/view/restaurant/l-s-kitchen/Projektor.rss?today=true&locale=#{locale}", location: "Lindholmen"},
+        {name: "Express", url: "http://intern.chalmerskonferens.se/view/restaurant/express/V%C3%A4nster.rss?today=true&locale=#{locale}"},
```

Comment:
The location for Express is Johanneberg. Not to be confused with "Dagens express" at Lindholmen.